sklearnex/_utils.py

```diff
@@ -63,3 +63,69 @@ def get_patch_message(s):
 def get_sklearnex_version(rule):
     from daal4py.sklearn._utils import daal_check_version
     return daal_check_version(rule)
+
+
+def get_patch_str(name=None, verbose=True):

```

Comment:
It's need when you have `get_patch_str_re`?

Reply:
get_patch_str need for write right patch lines in _distributor_init file
get_patch_str_re need to search our changes in _distributor_init